packages/sfc-playground/src/App.vue

```diff
@@ -32,6 +32,20 @@ const store = new ReplStore({
     : `${location.origin}/src/vue-server-renderer-dev-proxy`
 })
 
+let { mainFile } = store.state
+
+if (mainFile === 'App.vue') {
+  const firstFile = Object.keys(store.getFiles())[0]
+  if (['index.html', 'main.js', 'main.ts'].includes(firstFile)) {
+    mainFile = firstFile
+    store.setFiles(store.getFiles(), mainFile)
```

Comment:
`setFiles` is `async` and I'm not waiting for it here. This doesn't seem to matter, but the SSR check that follows this section needs to use `mainFile` and not `store.state.mainFile`, as the latter won't be updated yet.